doc/release/1.14.0-notes.rst

```diff
@@ -21,6 +21,12 @@ New functions
 Deprecations
 ============
 
+* Multidimensional indexing with anything but a base class tuple is
+  deprecated. This means that code such as ``arr[[slice(None)]]`` has to
```

Comment:
How about giving an example with a slightly more useful case, e.g., `[slice(None), slice(0, 10)]` -> `tuple(slice(None), slice(0, 10))` (or even omit the tuple in that case).